nussknacker-dist/src/universal/conf/dev-application.conf

```diff
@@ -73,6 +73,12 @@ environmentAlert: {
   cssClass: "indicator-green"
 }
 
+dynamicTabs = [
```

Comment:
I wonder if dynamicTabs is good name? They are pretty static ATM? Maybe customTabs? or sth?